### DISCUSSION TITLE
Not working on filament version 3

### DISCUSSION THREAD

**Discussion by dc-jahanzaib** [saade/filament-fullcalendar#105] [General]
Not working on filament version 3

**Comment by adriaardila** [comment]
It works if you specify this branch in composer.json: "saade/filament-fullcalendar": "3.x-dev",
It's still a WIP but it's working at least for me.
https://github.com/saade/filament-fullcalendar/pull/100

**Comment by saade** [comment]
use the v3.0.0-beta2 version
